### ISSUE TITLE
It doesn't appear that nfl rosters are updating

### ISSUE THREAD

**Issue by mill2121** [BurntSushi/nfldb#102]
I thought it was just a preseason issue at first, possibly waiting for the 53 man roster to be released.

For example, Ameer Abdullah (a rookie) is a player on the 53 man roster for Detroit. I don't see him anywhere in the players table. 

I ran nfl-update and still no Ameer. Any ideas?

**Comment by ochawkeye** [comment]
How are you trying to look him up?

```python
import nfldb
db = nfldb.connect()
print nfldb.player_search(db, "Ameer Abdullah")[0]
```
```
Ameer Abdullah (DET, RB)
```

```python
q = nfldb.Query(db)
q.game(season_year=2015, season_type='Regular')
q.player(full_name="Ameer Abdullah")
print q.as_aggregate()[0]
```
```
{'fumbles_tot': 1L, 'rushing_tds': 1L, 'receiving_yac_yds': 58L, 'fumbles_rec': 1L, 'kickret_yds': 105L, 'fumbles_notforced': 1L, 'receiving_rec': 4L, 'rushing_yds': 50L, 'receiving_yds': 44L, 'rushing_att': 7L, 'receiving_tar': 4L, 'kickret_ret': 3L}
```

**Comment by mill2121** [comment]
I'm actually not using any of the python functions, I am using just the nfldb database and writing the queries myself. And it doesn't appear he is located in my table.

I'm just trying a basic query, like: 
SELECT * FROM public.player WHERE first_name = 'Ameer';

**Comment by BurntSushi** [comment]
@mill2121 He's there for me:

```
[andrew@Liger ~] psql nfldb -c "SELECT full_name, team, position FROM player WHERE first_name = 'Ameer';"
   full_name    | team | position 
----------------+------+----------
 Ameer Abdullah | DET  | RB
(1 row)
```

Can you:

1. Run and show the output of `nflgame-update-players`?
2. Reset the last-player-update time, run `nfldb-update` and show the output?

You can reset and confirm the last-player-update time with this:

```
[andrew@Liger ~] psql nfldb -c "SET TIME ZONE 'UTC'; SELECT last_roster_download FROM meta;"
     last_roster_download      
-------------------------------
 2015-09-17 22:07:37.808378+00
[andrew@Liger ~] psql nfldb -c "SET TIME ZONE 'UTC'; UPDATE meta SET last_roster_download = '2000-01-01 00:00:00+00';"
UPDATE 1
[andrew@Liger ~] psql nfldb -c "SET TIME ZONE 'UTC'; SELECT last_roster_download FROM meta;"
  last_roster_download  
------------------------
 2000-01-01 00:00:00+00
[andrew@Liger ~] nfldb-update
```

**Comment by BurntSushi** [comment]
Also, try `nflgame-update-players --week 1`.

**Comment by mill2121** [comment]
Maybe I misunderstood how I use nfldb by itself. I'm not using nflgame at all, so I don't have the "nflgame-update-players" script. 

I always assumed that the "nfl-update" updated the rosters for me, but now I'm looking at nflgame and I notice 2 scripts that nfldb does not have (nflgame-update-players, nflgame-update-schedule). Do I need to set up nflgame in order to update my roster?

**Comment by BurntSushi** [comment]
> I'm not using nflgame at all,

If you're using `nfldb`, then you already have `nflgame` installed since `nflgame` is a [required dependency of `nfldb`](https://github.com/BurntSushi/nfldb/blob/master/setup.py#L17).

`nfldb-update` should be producing errors if it can't find the two nflgame scripts.

**Comment by mill2121** [comment]
Ahhh, ok. Sorry, I actually started my project with it a while ago and forgot that I had to install nflgame to make nfldb work. I just forgot about it because I haven't been using any of the nflgame python features since then, just using the database.

You were right, there was an error showing up when running nfldb-update, there was a permission error that it didn't have write access to the player database. I ran nfldb-update with "sudo" and it works fine now!

Thanks for putting together this project, it's made gathering nfl statistics really easy :)
